src/Sulu/Bundle/MediaBundle/Resources/config/image-formats.xml

```diff
@@ -15,37 +27,37 @@
         </commands>
     </format>
     <format>
-        <name>200x200</name>
+        <name>150x100</name>
         <commands>
             <command>
                 <action>scale</action>
                 <parameters>
-                    <parameter name="x">200</parameter>
-                    <parameter name="y">200</parameter>
+                    <parameter name="x">150</parameter>
+                    <parameter name="y">100</parameter>
                 </parameters>
             </command>
         </commands>
     </format>
     <format>
-        <name>50x50</name>
+        <name>100x100</name>
         <commands>
             <command>
                 <action>scale</action>
                 <parameters>
-                    <parameter name="x">50</parameter>
-                    <parameter name="y">50</parameter>
+                    <parameter name="x">100</parameter>
+                    <parameter name="y">100</parameter>
                 </parameters>
             </command>
         </commands>
     </format>
     <format>
-        <name>150x100</name>
+        <name>50x50</name>
         <commands>
             <command>
                 <action>scale</action>
                 <parameters>
-                    <parameter name="x">150</parameter>
-                    <parameter name="y">100</parameter>
+                    <parameter name="x">50</parameter>
+                    <parameter name="y">50</parameter>
                 </parameters>
             </command>
         </commands>
```

Comment:
@danrot no. All formats are the same i just added one and sorted them by their size. Github visualizes this in a confusing way